dask/array/tests/test_routines.py

```diff
@@ -1329,6 +1329,14 @@ def test_roll(chunks, shift, axis):
         assert_eq(np.roll(x, shift, axis), da.roll(a, shift, axis))
 
 
+def test_roll_bug():
```

Comment:
Can you change this to a more specific test name? `test_roll_bug` doesn't mean much to a reader, but `test_roll_always_results_in_a_new_array` does mean something.

```suggestion
def test_roll_always_results_in_a_new_array():
```

Reply:
Sure.